NetKet/Machine/abstract_machine.hpp

```diff
@@ -18,9 +18,9 @@
 #include <Eigen/Dense>
```

Comment:
This file only has formatting changes.

Reply:
Yes but the original file was not formatted correctly, so this is fine